Client/Info.plist

```diff
@@ -3,15 +3,15 @@
 <plist version="1.0">
 <dict>
 	<key>AdjustAppToken</key>
-	<string>$(ADJUST_APP_TOKEN)</string>
+	<string></string>
 	<key>AdjustEnvironment</key>
-	<string>$(ADJUST_ENVIRONMENT)</string>
```

Comment:
"buddybuilding" I like that 👍  "How do you guys do your Ci?" "I've got a buddy.. he builds it."